.travis.yml

```diff
@@ -40,14 +40,17 @@ addons:
 env:
   global:
     - ES2_DOWNLOAD_URL="https://download.elastic.co/elasticsearch/release/org/elasticsearch/distribution/tar/elasticsearch/2.4.5/elasticsearch-2.4.5.tar.gz"
+    - ES7_DOWNLOAD_URL="https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-7.0.1-linux-x86_64.tar.gz"
```

Comment:
v7.2.0 is how we roll around these places...

Reply:
We did it here: https://github.com/inveniosoftware/invenio-stats/pull/85